i6_models/parts/conformer/mhsa.py

```diff
@@ -60,3 +61,39 @@ def forward(self, input_tensor: torch.Tensor, sequence_mask: torch.Tensor) -> to
         output_tensor = torch.nn.functional.dropout(output_tensor, p=self.dropout, training=self.training)  # [B,T,F]
 
         return output_tensor
+
+
+class ConformerMHSAV2Config(ConformerMHSAV1Config):
+    pass
+
+
+class ConformerMHSAV2(torch.nn.Module):
+    """
+    Conformer multi-headed self-attention module
```

Comment:
Maybe mention on how this differs from V1

Reply:
Added.